.circleci/config.yml

```diff
@@ -651,3 +677,9 @@ workflows:
             branches:
               ignore:
                 - main
+  deploy_schemas:
+    jobs:
+      - schemas_deploy:
+          filters:
+            branches:
+              only: main
```

Comment:
This deploy could rely on the tests you wrote passing.

Reply:
Is it necessary to re-run the checks on main? We've already run them in the PR and merge queue, so not sure if this adds any value.